src/common/telemetry/src/logging.rs

```diff
@@ -206,8 +211,12 @@ pub fn init_global_logging(
 
         // Configure the file logging layer with rolling policy.
         let file_logging_layer = if !opts.dir.is_empty() {
-            let rolling_appender =
-                RollingFileAppender::new(Rotation::HOURLY, &opts.dir, "greptimedb");
+            let rolling_appender = RollingFileAppender::builder()
+                .rotation(Rotation::HOURLY)
+                .filename_prefix("greptimedb")
+                .max_log_files(opts.max_log_files)
+                .build(&opts.dir)
+                .expect("initializing rolling file appender failed");
```

Comment:
Maybe also carry log dir in panic info?

Reply:
> Better to return an error.
I simply add the dir info to it for now. Maybe we could do it in another pr .

Reply:
> > Better to return an error.
> 
> I simply add the dir info to it for now. Maybe we could do it in another pr .

@evenyag I think maybe it's reasonable to panic when log fails to initialize with given config instead of silently dropping all output.